### PR TITLE
Update Creator Store Docs

### PR DESCRIPTION
**Pull request by yusufameri**
## Checks

By submitting your pull request for review, you agree to the following:

- [ ] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [ ] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [ ] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/projects/assets/api.md [hunk 1]
```diff
@@ -1,195 +1,318 @@
...
 
-You can search Roblox's assets outside Studio by accessing the external catalog API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
+You can search Roblox's assets outside Studio by accessing the Creator Store API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
 
 Each API requires a URL and custom search parameters for that specific catalog. If both URL and parameters are valid, the API returns a JSON format with the results of your search.
 
 ## Creator Store API
 
+> [!NOTE]
+> We acknowledge that the toolbox-service API described below is not covered on the official Cloud APIs documentation. We are planning to better support Creator Store search with a cleaner API in future, stay tuned for updates.
```
Comment (IgnisRBX) [inline content/en-us/projects/assets/api.md:13]: Is there a specific reason why it's not being added to the official Cloud APIs doc? If so, it might be good to provide more specifics here instead of a promise for future updates.

Comment (yusufameri) [reply content/en-us/projects/assets/api.md:13]: It requires more planning to add to the official api docs. An internal team at roblox is working on getting a better estimate for this.

File: content/en-us/projects/assets/api.md [hunk 2]
```diff
@@ -1,195 +1,319 @@
...
+      <th>Type</th>
+      <th>Options and Values</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>categoryId (path)</td>
+      <td>int</td>
+      <td>
+        Audio = 3 </br>
```
[the remark this answers is not in the document]
Comment (yusufameri) [reply content/en-us/projects/assets/api.md:53]: @IgnisRBX I cant get these lint errors to go away? Am i missing something?

File: content/en-us/projects/assets/api.md [hunk 3]
```diff
@@ -1,195 +1,319 @@
...
 
-You can search Roblox's assets outside Studio by accessing the external catalog API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
+You can search Roblox's assets outside Studio by accessing the Creator Store API. Use the [Creator Store API](#creator-store-api) to query Studio assets, such as meshes, models, and audio, and the [Marketplace API](#marketplace-api) to query avatar assets on the Marketplace.
 
 Each API requires a URL and custom search parameters for that specific catalog. If both URL and parameters are valid, the API returns a JSON format with the results of your search.
 
 ## Creator Store API
 
+> [!NOTE]
+> We acknowledge that the toolbox-service API described below is not covered on the official Cloud APIs documentation. We are planning to better support Creator Store search with a cleaner API in future, stay tuned for updates.
```
Comment (IgnisRBX) [inline content/en-us/projects/assets/api.md:13]:
```suggestion
<Alert severity="info">
The `toolbox-service` API described below is not currently covered on the official [Cloud API](../../cloud/reference/index.md) documentation. Better support for Creator Store search with a cleaner API is underway.
</Alert>
```

Comment (IgnisRBX) [reply content/en-us/projects/assets/api.md:13]: I rephrased slightly and put into an Alert annotation here.

File: content/en-us/projects/assets/api.md [hunk 4]
```diff
@@ -1,195 +1,283 @@
...
+    <tr>
+      <td>maxPriceInCents</td>
+      <td>int64</td>
+      <td>Maximum cost in cents (only applicable to plugins)</td>
+    </tr>
+  </tbody>
 </table>
 
-The following URL will search for 10 items in the "Models" subcategory, sorted by most recently updated.
+For more detailed documentation regarding the toolbox-service, please refer to the updated documentation at creator store API.
```
Comment (IgnisRBX) [inline content/en-us/projects/assets/api.md:151]: Which documentation is this referring to?

File: content/en-us/projects/assets/api.md [hunk 5]
```diff
@@ -1,195 +1,283 @@
...
+
+- Audio = `3`
+- Model = `10`
+- Decal = `13`
+- Animation = `24`
+- Plugin = `38`
+- MeshPart = `40`
+- Video = `62`
+- FontFamily = `73`
+- Music = `300`
```
Comment (IgnisRBX) [inline content/en-us/projects/assets/api.md:31]: This could be a bit confusing, as there are 9 items listed here, then 13 items listed for `categoryId (path)` in the table below. Are these the same thing? Or this a subset of the tabled one, and what qualification makes it a subset?

File: content/en-us/projects/assets/api.md [hunk 6]
```diff
@@ -1,195 +1,283 @@
...
+      "id": int,
+      "name": string,
+      "searchResultSource": string
+    }
+  ]
+}
+```
 
-API responses return in a JSON format. The response provides asset details with the following primary fields:
+To get back the asset metadata, you would go over the data array and extract the ids, call the GetItemDetails endpoint on toolbox-service like this:
```
Comment (IgnisRBX) [inline content/en-us/projects/assets/api.md:181]: Is "GetItemDetails" a specific, proper API call or similar? Can we link to it elsewhere if so?

File: content/en-us/projects/assets/api.md (hunk 4 above)
Comment (yusufameri) [reply content/en-us/projects/assets/api.md:151]: I was referring to _this_ documentation that this PR added. Maybe I can reword this to 
> please refer to the updated documentation at creator store API below.

File: content/en-us/projects/assets/api.md (hunk 6 above)
Comment (yusufameri) [reply content/en-us/projects/assets/api.md:181]: Yeah, its a specific API. the example is given below, as mentioned. This API (toolbox-service) is a temporary workaround / API for creators until the creator store team builds a more robust API in Open Cloud, so these docs itself will be all that we give creators on how to use "search" in lieu of the catalog API deprecation.

File: content/en-us/projects/assets/api.md (hunk 4 above)
Comment (IgnisRBX) [inline content/en-us/projects/assets/api.md:151]:
```suggestion
For more details regarding `toolbox-service`, please refer to the updated documentation below.
```

File: content/en-us/projects/assets/api.md (hunk 5 above)
Comment (yusufameri) [reply content/en-us/projects/assets/api.md:31]: I updated it to only include the 9 items here and below

**Review by IgnisRBX — APPROVED** [review]
